srcpkgs/wine/template

```diff
@@ -152,16 +187,18 @@ wine-common_package() {
 	}
 }
 
-wine-tools_package() {
-	depends="wine-${version}_${revision}"
-	short_desc+=" - development tools"
-	if [ "${_nopie}" = yes ]; then
-		nopie=yes
-	fi
-	pkg_install() {
-		for file in widl winebuild winecpp winedump wineg++ winegcc winemaker wmc wrc function_grep.pl
-		do
-			vmove usr/bin/${file}
-		done
+if [ ! "$CROSS_BUILD" ] && [ ! $(vopt_if mingw true false)]; then
```

Comment:
do *not* optionally define a subpackage like this. use the `subpackages` variable